deps/wazuh_testing/wazuh_testing/tools/file.py

```diff
@@ -174,7 +176,7 @@ def delete_file(file_path):
 
 def delete_path_recursively(path):
     if os.path.exists(path):
-        shutil.rmtree(path)
+        shutil.rmtree(path, ignore_errors=True)
```

Comment:
Suppose we have a file in log directory that can not be deleted due to permission issues. So,
shutil.rmtree(dirPath, ignore_errors=True) will remove all the other files from ‘/somedir/logs’ directory except the file with permission issues. Also, it will not raise any error.

But thinking about  that here we don't have this problem.